server/main.go

```diff
@@ -84,6 +86,13 @@ func main() {
 	ble.SetDefaultDevice(device)
 	defer device.Stop()
 
+	if *enroll {
+		logrus.Info("Generating symmetric key")
+		if g_key, err = TPM2_GetRandom(32); err != nil {
```

Comment:
I agree it's better to be consistent, but keeping things on a single line is more compact and idiomatic go style. I'm not sure I'd have spotted the bug below with the split style for instance.

I suggest keeping this code as-is and a follow-up PR that would convert existing split style to single-line style.

What do you think?

Reply:
I personnally prefer the single-line version but there are some subtilities to consider because of how go assignation works:

```go
// The following creates somevar/err variables inside the if scope,
// meaning they doesn't change from the outer scope if they were declared.
if somevar, err := somefunc(); err != nil {
    ...
}

// This version updates outer variables, but they'll need to be declared before assignation
var somevar sometype
var err error

if somevar, err = somefunc(); err != nil {
    ...
}

// This one creates new variables but replaces previous ones with same name so that it's
// not necessary to declare them beforehand
err, somevar := somefunc()
if err != nil {
    ...
}
```